org.Cataclysm.DDA.Tiles.yml

```diff
@@ -0,0 +1,54 @@
+id: "org.Cataclysm.DDA.Tiles"
+runtime: "org.freedesktop.Platform"
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: /app/bin/launch-cataclysm-dda
+finish-args: 
+finish-args: 
+  [
+    "--filesystem=xdg-config/cataclysm-dda:create",
```

Comment:
it shouldn't need this permission

Reply:
this wasn't addressed

Reply:
This permission is required for the program to be able to create and view configuration folders and files in the user directory ~/.config/cataclysm-dda

Reply:
Are you sure? reading the launch script, it doesn't. And if it really needs it, it should be using `--persist` instead.

Reply:
I followed your advice, and the directory is not created, i still need to use `--filesystem`.

Reply:
What directory are you looking at. Given the value of `XDG_CONFIG_HOME` if you are looking at `.config` then you are looking in the wrong place.